docs/baking.md

```diff
@@ -0,0 +1,159 @@
+
+## Prerequisites
+
+### Installing required packages
+
+In order to run baking instance, you'll need the following Tezos binaries:
+`tezos-client`, `tezos-node`, `tezos-baker-<proto>`, `tezos-endorser-<proto>`.
+
+To install them run the following commands:
+```
+# Add PPA with Tezos binaries
+sudo add-apt-repository ppa:serokell/tezos
+sudo apt-get update
+# Install packages
+sudo apt-get install tezos-client tezos-node tezos-baker-<proto> tezos-endorser-<proto>
+```
+
+Packages for `tezos-node`, `tezos-baker-<proto>` and `tezos-endorser-<proto>` provide
+systemd units for running the corresponding binaries in the background
+
+### Downloading zcash-params
+
+Since v8.0 it's required to have `sapling-output.params` and `sapling-spend.params` files from
+the `zcash-params` within the `XDG_DATA_DIRS` to run the `tezos-node`.
+We'll need these params to be accessed by different
+users (because systemd services are usually run as a separate user), thus the suggested way of getting
+them via `https://raw.githubusercontent.com/zcash/zcash/master/zcutil/fetch-params.sh` script which will
+download them to the `HOME` directory of the current user is not convenient. Moreover, it will
+download large `sprout-groth16.params` which is unused at the runtime.
+So, it's better to download them manually to the `/usr/local/share/zcash-params`, in order
+to do that run the following commands:
+```
+sudo mkdir -p /usr/local/share/zcash-params
+sudo wget https://gitlab.com/tezos/opam-repository/-/raw/v8.1/zcash-params/sapling-output.params -O /usr/local/share/zcash-params/sapling-output.params
+sudo wget https://gitlab.com/tezos/opam-repository/-/raw/v8.1/zcash-params/sapling-spend.params -O /usr/local/share/zcash-params/sapling-spend.params
+```
+
+## Setting up the tezos-node
+
+### Bootstrapping the node
+
+In order to run baker locally, you'll need fully-synced local `tezos-node`.
+The easiest way to set up a node running on the `mainnet` or on one of the
+testnets (e.g. `delphinet` or `edonet`) is to use one of the predefined
+`tezos-node-<network>.services` systemd services provided in the `tezos-node`
+package. However, by default, these services will start to bootstrap the node
+from scratch, which will take significant amount of time. The fastest way to bootstrap
+the node is to import a snapshot. The snapshots can be downloaded from the https://snapshots.tulip.tools/#/ .
+
+`tezos-node-<network>.service` has `/var/lib/tezos/node-<network>` as a data directory
+and `http://localhost:8732` as a an RPC address by default.
+These parameters can be changed in the service configuration, which is located in
+`/lib/systemd/system/tezos-node-<network>.service`. Note, that if you've updated the `.service`
+file, you should reload systemctl daemon via running `sudo systemctl daemon-reload`.
+
+All commands within the service are run under `tezos` user.
+
+Download the snapshost for desired network and with desired history mode
```

Comment:
```suggestion
Download the snapshost for desired network and with the desired history mode
```

Reply:
One more thing: it's a bit odd that the snapshots and were to find them is separated to this sentence by the part explaining the dir and parameter of the node.

Perhaps this section that's now in the middle can be moved above (just before the sentences that mention snapshots)? Or maybe it would be better to move it just below this?

Reply:
Yes, I think that the paragraph that starts from '`tezos-node-<network>.service` has `/var/lib/tezos/node-<network>` as a data directory' can be moved to the very top of the 'Setting up the tezos-node' part, right before the 'Bootstrapping the node'